### PR TITLE
Adds .gitignore to drupal checkout

### PR DESCRIPTION
**Pull request by mradcliffe**
Not sure if the example.gitignore file needs a separate directory to be stored in this repository or not.

### CODE REVIEW

**Review by rickmanelius — APPROVED** [review]
LGTM +1

**Comment by rfay** [comment]
@mradcliffe Could you please sign the CLA? Thanks!

**Review by rfay — COMMENTED** [review]
Isn't this going to make them accidentally add the gitignore to their patch in the sprint?

**Comment by rickmanelius** [comment]
@rfay The gitignore is gitignore'd! :) https://github.com/mradcliffe/quicksprint/blob/e46dee8e6cc2d330dfe0294df09cc6283be4accf/example.gitignore#L32

**Comment by mradcliffe** [comment]
Oh, I know why, I mispelled my email address in my git config. How shameful. :(

**Review by rfay — COMMENTED** [review]
Questions:
* Should .ddev be added to the gitignore?
* Should .db_dumps be added?

**Comment by mradcliffe** [comment]
Yes, I think that it should too. 👍

**Comment by rfay** [comment]
If you push an update, please add /.phantomjs in there as well.

File: example.gitignore [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# */private
+
+# Ignore SimpleTest multi-site environment.
+# simpletest
+
+# Sprint tools
+.ddev/
+.db_dumps/
+.thelounge/
+.phantomjs/
```
Comment (mradcliffe) [inline example.gitignore:65]: I was trying to find docs on whether this is a file or directory, but wasn't sure.

Comment (BrianGilbert) [reply example.gitignore]: @mradcliffe needs .c9 as well please

File: example.gitignore [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+# Because .gitignore can be specific to your site, this file has a different
+# name; updating Drupal core will not override your custom .gitignore file.
+
+# Ignore core when managing all of a project's dependencies with Composer
+# including Drupal core.
+# core
+
+# IDEs
+.idea/
+.c9/
```
Comment (mradcliffe) [inline example.gitignore:16]: It's up here in the IDE section.

**Comment by rfay** [comment]
I'm good with this whenever you think it's ready to pull, but it needs rebase.

**Pull request merged**